src/main/java/com/oauth/server/api/ReciprocalAuthorizationEndpoint.java

```diff
@@ -77,6 +93,11 @@ public void postReciprocalCode(final @RequestBody @RequestParam Map<String, Stri
             createAccessTokenRequest(authorizationCode));
 
         partnerTokenRepository.saveAccessToken(resourceDetails, auth, accessToken);
+
+        // Baby profiles corresponding to a user must be published to Alexa during account linking and when the
+        // profile changes after account linking. Following code publishes the profiles when customer enables the
+        // skill and does account linking. To publish profiles outside of account linking, use publish profiles endpoint.
+        babyActivityProfilePublisher.publishProfiles(clientAccessToken, partnerId);
```

Comment:
This should probably be a sync action as the failure of profile publishing should not block reciprocal authorization process.

Reply:
done